config/dev.exs

```diff
@@ -3,7 +3,7 @@ import Config
 # Configure your database
 config :tarragon, Tarragon.Repo,
   username: "postgres",
-  password: "postgres",
+  password: "welcome2",
```

Comment:
Please remove this file from the change.

You can reset one file with the following command:

From project root
`git checkout main ./config/dev.exs`